cmd/admin-revoker/main.go

```diff
@@ -126,12 +132,7 @@ func revokeBySerial(serial string, reasonCode int, deny bool, cac rpc.Certificat
 	return
 }
 
-func revokeByReg(regID int, reasonCode int, deny bool, cac rpc.CertificateAuthorityClient, auditlogger *blog.AuditLogger, tx *gorp.Transaction) (err error) {
-	_, err = tx.Get(core.Registration{}, regID)
-	if err != nil {
-		return
-	}
-
+func revokeByReg(regID int64, reasonCode int, deny bool, cac rpc.CertificateAuthorityClient, auditlogger *blog.AuditLogger, tx *gorp.Transaction) (err error) {
```

Comment:
for those of you squinting at this: regID's type was int and needs to be int64 to talk over the RPC.

See the parseInt change below to enable that.